make_queue/forms.py

```diff
@@ -202,6 +202,17 @@ def __init__(self, *args, **kwargs):
             widget=SemanticSearchableChoiceInput(attrs={"required": True}),
         )
 
+    def clean(self):
+        cleaned_data = super().clean()
+
+        machine_type = cleaned_data.get("machine_type")
+
+        if machine_type is not None and machine_type.has_stream and cleaned_data["stream_name"] is None:
+            self._errors['stream_name'] = [_("Stream Name cannot be empty")]
+            raise ValidationError(_("stream_name cannot be None"), code="stream_name_is_none")
```

Comment:
Not sure if this is best practice, but without line 211, there was no error message when submitting the form. There is also a different message, as a user may not understand the `None` term.

Reply:
To also add the `code` kwarg to the exception (and also follow Django's preference for using `ValidationError` objects), you can do this: 😊 
```python
raise forms.ValidationError({
    'stream_name': forms.ValidationError(_("Stream name cannot be empty"), code='stream_name_is_none'),
})
```

Reply:
Fixed with `self.add_error`